packages/requisitions/src/SupplierRequisition/api/api.ts

```diff
@@ -30,19 +30,22 @@ export const requisitionToInput = (
 export const RequestRequisitionQueries = {
   get: {
     list:
-      (api: RequestRequisitionApi) =>
+      (api: RequestRequisitionApi, storeId: string) =>
```

Comment:
Adding storeID as a parameter and adding to queries replacing the hard coded store ID I added